tests/kafkatest/services/templates/mirror_maker_consumer.properties

```diff
@@ -27,3 +29,9 @@ offsets.storage={{ offsets_storage }}
 {% if consumer_timeout_ms is defined and consumer_timeout_ms is not none %}
 consumer.timeout.ms={{ consumer_timeout_ms }}
 {% endif %}
+
+{% if new_consumer %}
```

Comment:
This change seems unrelated to the patch? And changes the defaults such that no tests would cover the default MM settings?